pkg/api/v0/workload.go

```diff
@@ -59,19 +59,6 @@ type WorkloadInstance struct {
 	// All events generated for the workload instance that aren't related to a
 	// particular workload resource instance.
 	Events []*WorkloadEvent `json:"Events,omitempty" query:"events" validate:"optional"`
-
-	// The threeport objects that are deployed to support the workload instance.
-	AttachedObjectReferences []*AttachedObjectReference `json:"AttachedObjectReferences,omitempty" query:"attachedobjectreferences" validate:"optional,association"`
-}
-
-// AttachedObjectReference is a reference to an attached object.
-type AttachedObjectReference struct {
```

Comment:
Is this reintroduced elsewhere?

Reply:
Yes, it now lives in `pkg/api/v0/attached_object.go`